cmd/flags.go

```diff
@@ -259,3 +261,15 @@ func retryFlag(cmd *cobra.Command) *cobra.Command {
 	}
 	return cmd
 }
+
+func updateTimeFlags(cmd *cobra.Command) *cobra.Command {
+	cmd.Flags().DurationP(flagTimeInterval, "i", 60*time.Second, "time interval to run update client process")
```

Comment:
lets make the default larger. I suspect trusting periods will likely be greater than 1 week, since we update clients before relaying packets, we only need this to prevent clients from becoming frozen. Maybe something like 1 hour would be safe?

cc @AdityaSripal